exercises/binary-search-tree/src/test/java/BinarySearchTreeTest.java

```diff
@@ -108,10 +108,10 @@ public void createsComplexTree() {
     public void sortsSingleElement() {
```

Comment:
I still think the test above this one (`createsComplexTree`) doesn't quite match the canonical data 🙂

Reply:
@sjwarner-bp